### ISSUE TITLE
`libRCTAlipay.a` binary build failed, need update

### ISSUE THREAD

**Issue by EvianZhow** [0x5e/react-native-alipay#9]
```
undef: _OBJC_CLASS_$_UTDevice
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_UTDevice", referenced from:
      objc-class-ref in libRCTAlipay.a(libAlipaySDK.a)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Fixed according to these steps: https://my.oschina.net/u/1583729/blog/790413

Newer binary MD5:
```
openssl md5 /Users/_/Downloads/WS_APP_PAY_SDK_BASE_2.0/Demo/iOSDemo_2.0\(SDK_15.5.3\)/AlipaySDK.framework/AlipaySDK
MD5(/Users/_/Downloads/WS_APP_PAY_SDK_BASE_2.0/Demo/iOSDemo_2.0(SDK_15.5.3)/AlipaySDK.framework/AlipaySDK)= c49fc1c0db8945f084506af5902abc7a
```

**Comment by EvianZhow** [comment]
FYI: develop branch

**Comment by 0x5e** [comment]
Merged #10
